prow/plugins.yaml

```diff
@@ -193,3 +193,12 @@ plugins:
   - owners-label
   - size
   - trigger
+  GoogleCloudPlatform/oss-test-infra:
```

Comment:
Its gross that we're copying the plugin config like this. I'm assuming this is because we are receiving webhooks for the entire GoogleCloudPlatform org, not just the repos we care about?

Reply:
good point - we receive webhooks for individual repos, will update